AWS/legos/aws_ebs_modify_volume/aws_ebs_modify_volume.py

```diff
@@ -80,7 +80,11 @@ def aws_ebs_modify_volume(
         newSize = currentSize * resize_value
 
     print(f'CurrentSize {currentSize}, NewSize {newSize}')
-
+    
+    ec2Client.modify_volume(
+        VolumeId=volume_id,
+        Size=newSize) 
+    

```

Comment:
:-)

Thanks for looking into this.
The [response](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2/client/modify_volume.html) is a object with the following syntax:

```
{
    'VolumeModification': {
        'VolumeId': 'string',
        'ModificationState': 'modifying'|'optimizing'|'completed'|'failed',
        'StatusMessage': 'string',
        'TargetSize': 123,
        'TargetIops': 123,
        'TargetVolumeType': 'standard'|'io1'|'io2'|'gp2'|'sc1'|'st1'|'gp3',
        'TargetThroughput': 123,
        'TargetMultiAttachEnabled': True|False,
        'OriginalSize': 123,
        'OriginalIops': 123,
        'OriginalVolumeType': 'standard'|'io1'|'io2'|'gp2'|'sc1'|'st1'|'gp3',
        'OriginalThroughput': 123,
        'OriginalMultiAttachEnabled': True|False,
        'Progress': 123,
        'StartTime': datetime(2015, 1, 1),
        'EndTime': datetime(2015, 1, 1)
    }
}
```

It would be good to use `resp` and print out the `StatusMessage` rather than dropping it altogether .

Thanks for looking into this.

Reply:
I have added the print statement